spec/lib/import_csv_to_big_query_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'import_csv_to_big_query'
+require 'rails_helper'
+
+RSpec.describe ImportCSVToBigQuery do
+  let(:table) { double('table').as_null_object }
+  let(:loaded_job) { double('loaded job').as_null_object }
+  let(:dataset) { double('dataset').as_null_object }
+  let(:bigquery_client) { double('Bigquery Client', dataset: dataset) }
+  let(:export_tables) { class_double(ExportTablesToCloudStorage, TABLES: %w[TestTable]) }
+
+  it 'loads the dataset from ENV' do
```

Comment:
Does this mean 'dataset name' or 'dataset'?

Reply:
dataset in big query - this might look something like `staging_tvs_dataset`

Reply:
The dataset in big query

Reply:
Which is just a collection of tables etc. in big query